### DISCUSSION TITLE
Menu option to open preferences directory

### DISCUSSION THREAD

**Discussion by talmo** [talmolab/sleap#1830] [Ideas]
The user preferences get saved in a user data directory that's OS dependent. It would be handy to have an option (under Help? File?) to open the system File Explorer at that directory so people don't have to dig around their system paths to find it.

**Comment by gitttt-1234** [comment]
Note: Also update these routines that handle management of the preferences directory to use `pathlib`, and break it out into its own methods:

https://github.com/talmolab/sleap/blob/05283629a67385f40a6e8302519393fb2f87ecfb/sleap/util.py#L276-L289

**Comment by talmo** [comment]
This was done in #1847 --> #1843
